projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,8 @@
+<cdk-virtual-scroll-viewport appendOnly itemSize="10" class="tabs-viewport">
+    <div class="virtual" *cdkVirtualFor="let c of allColors; let i = index" class="tabs-item">
+        <geoengine-color-attribute-input [ngModel]="c" (ngModelChange)="rebuildColorMap(i, $event)"></geoengine-color-attribute-input>
+        <button mat-mini-fab (click)="removeColorTab(i)" class="removeButton">-</button>
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/225276004-a6c602c3-1825-4a7f-b9b1-f79b1cdc2385.png)

Please make this more subtle, e.g.

```suggestion
        <button mat-icon-button (click)="removeColorTab(i)" class="removeButton"><mat-icon>remove</mat-icon></button>
```